mods/cnc/weapons/superweapons.yaml

```diff
@@ -16,8 +16,8 @@ Atomic:
 		Explosions: nuke_explosion
 		ImpactSounds: nukexplo.aud
 	Warhead@3Dam_areanukea: SpreadDamage
-		Spread: 2c512
-		Damage: 100
+		Spread: 5c512
```

Comment:
This increases the effective area of effect to a whopping 66(!) cell diameter, damaging pretty much everything on smaller maps like Avocado.

A flatter Falloff (f.ex. `1000, 700, 500, 300, 150, 50, 0`) would be a much better approach, considering this is mostly about damage in the nearer vicinity of the impact.